src/main/java/frc/robot/autons/Auton.java

```diff
@@ -1,14 +1,16 @@
 package frc.robot.autons;
 
+import java.util.Optional;
+
 import edu.wpi.first.wpilibj.DriverStation.Alliance;
 import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
 import frc.robot.subsystems.DriveTrain;
 
 public abstract class Auton extends SequentialCommandGroup {
     protected DriveTrain driveTrain = DriveTrain.getInstance();
-    protected Alliance alliance;
+    protected Optional<Alliance> alliance;
 
-    public Auton(Alliance alliance) {
-        this.alliance = alliance;
+    public Auton(Optional<Alliance> alliance2) {
```

Comment:
This change has been reject more than once. I’m not sure which branch has the permanent fix to this but please look around for it. Code should look like `if (alliance.isPresent())  {this.alliance = alliance }`